app/src/main/kotlin/com/simplemobiletools/gallery/pro/dialogs/ChangeFileThumbnailStyleDialog.kt

```diff
@@ -1,11 +1,14 @@
 package com.simplemobiletools.gallery.pro.dialogs
 
 import android.content.DialogInterface
+import android.widget.Toast
```

Comment:
this is no longer needed, always optimize imports and code style before the PR

Reply:
Sorry, I am still getting used to IntelliJ. Did it now for all changed files.